models/serializers/DataSerializer.cfc

```diff
@@ -18,14 +18,39 @@ component singleton {
     }
 
     /**
+    * Decides how to nest the data under the given identifier.
+    *
+    * @data       The serialized data.
+    * @identifier The current identifier for the serialization process.
+    *
+    * @returns    The scoped, serialized data.
+    */
+    function scopeData( data, identifier ) {
+        return { "#listLast( identifier, "." )#" = { "data" = data } };
```

Comment:
Should an init method be added on this one as well, allowing a `rootName` value to be set explicitly?